docker/image_types.yaml

```diff
@@ -28,6 +28,9 @@
   tag: 'builders/coverage/{benchmark}'
```

Comment:
Probably, the name and tag should add `fuzzer` too.

Reply:
discussed offline